docs/RELEASE.md

```diff
@@ -2,39 +2,38 @@
 
 ## Release candidate
 1. Create a release task in GitHub using the "New Release" template
-2. Create and push the `release` branch
-3. Update APP_VERSION_NAME (x.y.z)
-4. Build pipeline will produce and upload builds with `rc` prefix (x.y.z-buildNumber`rc`-buildVariant)
-5. Notify QA about release candidate
-6. Create pull-request for release branch in case additional work/fixes are required for release candidate and go to step 4.
-7. Normal development continues targeting `master` branch.
+1. Create and push the `release` branch
```

Comment:
every step is 1. Also, it's not clear what happened to other items -> everything seems to be replaced which is not the case. It's hard to review with such formatting issues